apisix/cli/ngx_tpl.lua

```diff
@@ -19,6 +19,8 @@ return [=[
 # Configuration File - Nginx Server Configs
 # This is a read-only file, do not try to modify it.
 
+user root;
```

Comment:
It's dangerous to use `root` to launch APISIX. Exposing this item to `config.yaml` so people can decide to use which user.

Reply:
This can be archived without modifying APISIX: https://github.com/apache/apisix/blob/master/docs/en/latest/customize-nginx-configuration.md

Reply:
I have add  a configuration field in `config-default.yaml`,and add `How to run APISIX on the Chinese OS` to FAQ.